rkt/stop.go

```diff
@@ -37,52 +37,67 @@ var (
 func init() {
 	cmdRkt.AddCommand(cmdStop)
 	cmdStop.Flags().BoolVar(&flagForce, "force", false, "forced stopping")
+	cmdStop.Flags().StringVar(&flagUUIDFile, "uuid-file", "", "read pod UUID from file instead of argument")
 }
 
 func runStop(cmd *cobra.Command, args []string) (exit int) {
 	var podUUID *types.UUID
 	var podUUIDs []*types.UUID
+	var err error
 	var errors int
 
-	if len(args) < 1 {
-		cmd.Usage()
-		return 1
-	}
-
-	for _, uuid := range args {
-		podUUID, err := resolveUUID(uuid)
+	ret := 0
+	switch {
+	case len(args) == 0 && flagUUIDFile != "":
```

Comment:
So it is either `UUID` or `--uuid-file=FILE`? Why not both? In this case, the usage in cobra should be updated.
```
Use:   "stop --uuid-file=FILE | UUID ...",
```